Classes/Command/CrawlerCommandController.php

```diff
@@ -76,8 +80,16 @@ public function crawlSitesCommand(): void {
             }
 
             $urlSchemeAndHost = (string)$domain;
-            $this->outputLine('Crawling site <b>"%s"</b> with urlSchemeAndHost <i>"%s"</i>', [$siteNodeName, $urlSchemeAndHost]);
-            $this->crawlNodesCommand($siteNodeName, $urlSchemeAndHost);
+
+            if($method == 'robotstxt'){
+                $urlSchemeAndHost = $scheme.'://'.$urlSchemeAndHost.'/robots.txt';
```

Comment:
```suggestion
                $urlSchemeAndHost = $urlSchemeAndHost.'/robots.txt';
```

Reply:
This wont work as crawlRobotsTxt() needs an URL with Scheme (http/s) @Sebobo

Reply:
Then you need to check whether the domain has already a record, because if you have on defined there you have two ;)
Also check my latest commit in which I allow enforcing a scheme.